packages/react-examples/.storybook/main.js

```diff
@@ -48,4 +41,19 @@ const config = {
   },
 };
 
+function getStories() {
+  const packageName = path.basename(__dirname);
+
+  if (!fs.existsSync(path.resolve(__dirname, '../src', packageName))) {
+    throw new Error(`Package ${packageName} does not have examples!`);
+  }
+
+  // For @fluentui/react's storybook, also include examples from @fluentui/react-focus
+  if (packageName === 'react') {
```

Comment:
this came from https://github.com/microsoft/fluentui/pull/32074/files#diff-e11bfa97318c042abbf9cd46418de9afda2274a5e8d873e200ee2a5e8bec9832L64